nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -10,7 +10,7 @@ You can use the Node Maintenance Operator to place nodes in maintenance mode. Th
 
 [NOTE]
 ====
-If you have installed {VirtProductName}, you must use the Node Maintenance Operator that is bundled with it.
+If {VirtProductName} v4.10.z or less is installed in your cluster, it includes an outdated version of the Node Maintenance Operator version. Upgrade {VirtProductName} to avail of the latest version of the Node Maintenance Operator.
```

Comment:
s/avail of/obtain/

Reply:
Thanks John, I've included this feedback.